lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
 
-const BbPromise = require('bluebird');
 const _ = require('lodash');
+const BbPromise = require('bluebird');
+const pathLib = require('path');
+
+const naming = require(pathLib.join(__dirname, '..', '..', '..', '..', '..', 'lib', 'naming.js'));
```

Comment:
Oh...  I misread this initially.  I've removed the painful use of path.  There was a set of cases where my search had failed to catch some occurrences.

Reading you as requesting that naming.js be moved into `~/lib/plugins/aws/deploy/compile/events/apiGateway/index.js`, I have to note that the naming module is defined at `~/lib/plugins/aws/lib/naming.js` because the naming conventions need to be applied through the provider specific plugin.  Examples include:
`~/lib/plugins/aws/index.js`
`~/lib/plugins/aws/info/index.js`
`~/lib/plugins/aws/logs/index.js`
`~/lib/plugins/aws/remove/index.js`